sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs

```diff
@@ -259,7 +259,33 @@ public virtual ServiceBusSender CreateSender(string queueOrTopicName)
 
             return new ServiceBusSender(
                 entityPath: queueOrTopicName,
```

Comment:
To avoid duplication, should we consider delegating the implementation to the overload and just pass `null`  or `new()` for the `options`?